oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/VersionGarbageCollectorIT.java

```diff
@@ -270,7 +270,7 @@ public void tearDown() throws Exception {
         execService.shutdown();
         execService.awaitTermination(1, MINUTES);
         fixture.dispose();
-        LOG.info("tearDown: START. fullGcMode = {}, fixture = {}", fullGcMode, fixture);
+        LOG.info("tearDown: DONE. fullGcMode = {}, fixture = {}", fullGcMode, fixture);
```

Comment:
might be better to do this in a separate... Oh well, doesn't matter :-)

Reply:
it was sneaky, I know, won't happen again :)